### PR TITLE
KAFKA-5965: Remove Deprecated AdminClient from Streams Resetter Tool

### PR DESCRIPTION
**Pull request by fedosov-alexander**
Removed usage of deprecated AdminClient from StreamsResetter
No additional tests are required.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by mjsax** [comment]
\cc @bbejeck @vvcephei 

@fedosov-alexander Thanks for the PR -- test failure seems to be related. We can review after those a fixed. Let us know if  you need any input.

Btw: you can trigger a new build, by putting a comment "Retest this please" on Github. Not need to do a "dummy commit"

File: core/src/main/scala/kafka/tools/StreamsResetter.java [hunk 1]
```diff
@@ -149,18 +153,12 @@ public int run(final String[] args,
...
-            if (!olderAdminClient.describeConsumerGroup(groupId, 0).consumers().get().isEmpty()) {
-                throw new IllegalStateException("Consumer group '" + groupId + "' is still active. "
-                                                + "Make sure to stop all running application instances before running the reset tool.");
-            }
-        } finally {
-            if (olderAdminClient != null) {
-                olderAdminClient.close();
-            }
+                                           final AdminClient adminClient) throws ExecutionException, InterruptedException {
+        DescribeConsumerGroupsResult describeResult = adminClient.describeConsumerGroups(Arrays.asList(groupId),
```
Comment (mjsax) [inline core/src/main/scala/kafka/tools/StreamsResetter.java]: nit: add `final` (we use `final` whenever possible)

**Review by mjsax — APPROVED** [review]
Thanks for updating the PR.

Just one nit. Otherwise LGTM.

Call for second review @guozhangwang @bbejeck @vvcephei

File: core/src/main/scala/kafka/tools/StreamsResetter.java [hunk 2]
```diff
@@ -149,18 +153,12 @@ public int run(final String[] args,
...
+                (new DescribeConsumerGroupsOptions()).timeoutMs(10 * 1000));
+        if (!describeResult.describedGroups().get(groupId).get().members().isEmpty()) { 
+            throw new IllegalStateException("Consumer group '" + groupId + "' is still active. "
```
Comment (guozhangwang) [inline core/src/main/scala/kafka/tools/StreamsResetter.java]: nit: list the members list here would help trouble shoot.

**Comment by fedosov-alexander** [comment]
Dear @guozhangwang @bbejeck @vvcephei thank you for your recommendations.
 I've made requested changes, please,review.

**Review by guozhangwang — APPROVED** [review]

**Comment by guozhangwang** [comment]
Thanks @fedosov-alexander ! merged to trunk.

**Pull request merged**